ic-utils/src/interfaces/http_request.rs

```diff
@@ -29,6 +29,18 @@ pub struct HttpResponse {
     pub body: Vec<u8>,
 }
 
+#[derive(CandidType)]
+pub struct HttpGetChunkRequest {
```

Comment:
can this be moved inside the body of `http_get_chunk` similar to https://github.com/dfinity/agent-rs/blob/next/ic-utils/src/interfaces/management_canister.rs#L453